portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -96,4 +124,20 @@ private int numComments(String number) {
     }
     return commentNumber;
   }
+
+   /** Returns the nickname of the user with id, or null if the user has not set a nickname. */
+  private String getUserNickname(String id) {
+
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I think 131 and 132 need to be indented.

Reply:
solved